### DISCUSSION TITLE
Pandas dataframe attachment - index=False?

### DISCUSSION THREAD

**Discussion by luisdelatorre012** [Miksus/red-mail#61] [Q&A]
When I pass a pandas dataframe as an attachment, is there a way to have it save the dataframe without adding the index column?

I can do this with:

```
    attachment_file = pathlib.Path(filename)
    my_data_frame.to_csv(attachment_file, index=False)

    send_email(
        ...,
        attachments={attachment_file.name: attachment_file},
    )
```

but is there something like this?

```
    send_email(
        ...,
        attachments={filename: my_data_frame},
        attachments_kwargs={"index":False}
    )
```

**Comment by ejbills** [comment]
I have a PR open currently that adds support for this using the pandas Stylers objects. It can be accomplished by something like this:

https://pandas.pydata.org/docs/reference/api/pandas.io.formats.style.Styler.hide_index.html

See my PR comments for an example of this in code. 

In the meantime, you can clone my fork of redmail and use it locally.
